staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,56 +44,57 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// Parallelism defines the amount of parallelism in algorithms for scheduling a Pods. Must be greater than 0. Defaults to 16
+	// The amount of parallelism in algorithms for scheduling Pods.
+	// Must be greater than 0. Defaults to 16.
 	Parallelism *int32 `json:"parallelism,omitempty"`
 
-	// LeaderElection defines the configuration of leader election client.
+	// The configuration of leader election client.
 	LeaderElection componentbaseconfigv1alpha1.LeaderElectionConfiguration `json:"leaderElection"`
 
-	// ClientConnection specifies the kubeconfig file and client connection
-	// settings for the proxy server to use when communicating with the apiserver.
+	// This specifies the kubeconfig file and client connection settings for the scheduler
+	// when communicating with the API server.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
-	// HealthzBindAddress is the IP address and port for the health check server to serve on,
-	// defaulting to 0.0.0.0:10251
+	// The IP address and port for the health check server to serve on,
```

Comment:
Users can just specify an IP address (no port) and the scheduler's defaults will automatically append the default port. We should mention this undocumented feature here.

Reply:
correction: we want to remove this hack, so please ignore the above suggestion